cg/server/ext.py

```diff
@@ -32,6 +36,13 @@ def init_app(self, app):
         super(FlaskStore, self).__init__(uri)
 
 
+class CustomJSONEncoder(JSONEncoder):
```

Comment:
What happens if this is not handled?

Reply:
Most fields are serialisable by default by the `JSONEncoder`, but the `Decimal` (`BigInteger`) field in the sequencing metrics table was an exception. Adding the same handling to all places where we might have a `Decimal` seemed annoying, so I thought it would be better to handle the fields which are not serialisable by default here

Reply:
But yeah, if a field cannot be serialised we get an internal server error when the endpoint tries to serialise the database object to json. There are some other solutions to this problem, such as using a json serialisation library like `Marshmallow` in Python. That would require us to add separate schema models for all our database objects that we return and serialise in the API

Reply:
Which seemed out of scope for this PR 😅 Not sure how we want to do this, but this seems like an OK way for now

Reply:
Thanks for the explanation. I think this implementation looks good 👍